### PR TITLE
Add Lego for https management.

### PR DESCRIPTION
**Pull request by brendandburns**
@spxtr @fejta @rmmh @eparis 

This is already deployed:

https://kubernetes.submit-queue.k8s.io/#/queue

### CODE REVIEW

File: mungegithub/submit-queue/ingress.yaml [hunk 1]
```diff
@@ -28,3 +31,10 @@ spec:
         - backend:
             serviceName: test-infra-sq-status
             servicePort: 80
```
Comment (rmmh) [inline mungegithub/submit-queue/ingress.yaml]: There should be an entry here for host: submit-queue.k8s.io

It looks like it's currently not behind a LoadBalancer Ingress, so the DNS will have to be updated as well.

**Comment by rmmh** [comment]
There are a few more HTTP references here:

```
submit-queue/deployment/kubernetes/configmap.yaml
119:  submit-queue.chart-url: http://storage.googleapis.com/kubernetes-test-history/k8s-queue-health.svg
120:  submit-queue.history-url: http://storage.googleapis.com/kubernetes-test-history/static/index.html
```

**Comment by eparis** [comment]
since he already deployed this to production i'm going to merge. @brendanburns can you address his last 2 comments in another PR?

File: mungegithub/submit-queue/ingress.yaml (hunk 1 above)
Comment (brendandburns) [inline mungegithub/submit-queue/ingress.yaml]: Done, though I don't have access to update DNS, so someone else will have to do that after this is deployed.

**Comment by brendandburns** [comment]
Comments addressed in a follow-on PR.

**Pull request merged**